css/style.scss

```diff
@@ -5,15 +5,17 @@
 $image-path: '../../img';
 $font-path: '../../fonts';
 
-@import '@mozilla-protocol/core/protocol/css/protocol';
-@import '@mozilla-protocol/core/protocol/css/includes/lib';
-@import '@mozilla-protocol/core/protocol/css/components/button';
-@import '@mozilla-protocol/core/protocol/css/templates/multi-column';
-@import '@mozilla-protocol/core/protocol/css/components/footer';
+@import '~@mozilla-protocol/core/protocol/css/protocol';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/button';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import '~@mozilla-protocol/core/protocol/css/components/footer';
 @import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
 @import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-mozilla';
-
-@import "footer";
+@import '~@mozilla-protocol/core/protocol/css/components/forms/button-container';
+@import '~@mozilla-protocol/core/protocol/css/components/forms/choice';
+@import '~@mozilla-protocol/core/protocol/css/components/forms/field';
+@import '~@mozilla-protocol/core/protocol/css/components/forms/form';
```

Comment:
What's the `~` for the `@import`s do?

Reply:
oh I don't think it's actually necessary for @import, the tilde seems to be a thing for @use... I'll remove